code/modules/research/nanites/nanite_programs/buffing.dm

```diff
@@ -44,15 +44,13 @@
 	. = ..()
 	if(ishuman(host_mob))
 		var/mob/living/carbon/human/H = host_mob
-		H.physiology.armor.melee += 30
-		H.physiology.armor.bullet += 30
+		H.physiology.physio_armor.generate_new_with_specific(list(MELEE = 30, BULLET = 30))
 
 /datum/nanite_program/hardening/disable_passive_effect()
 	. = ..()
 	if(ishuman(host_mob))
 		var/mob/living/carbon/human/H = host_mob
-		H.physiology.armor.melee -= 30
-		H.physiology.armor.bullet -= 30
+		H.physiology.physio_armor = null
 
```

Comment:
This won't stack at all with other things trying to modify physio armour, essentially making it a nanite buffing armour only. Either find a way to do this that can work cross-modules, or find some other replacement effect for modifying the physiology armour.

Reply:
I hate physiology armor who added that shit

Reply:
Thoughts? 
b35451e